### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by alutman-innablr**
Entry point for revolver development.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Contributing to Revolver
+
+## Structure
+Revolver works with a pluggable interface for `drivers` and `plugins` that are executed per cloud account.
+* `drivers` implement cloud API calls, both query and update.
+* `plugins` implement power cycling logic.
```
Comment (lyricnz) [inline CONTRIBUTING.md]: Technically power-cycling is only 1-2 plugins, plus tag validation. May be extended by other plugins.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+npm run build # or 'npm run watch' for a continuous build
+npm run start
+```
+
+`start` utilizes `invoke.ts/js` to call the true `revolver.ts` with an event. This is necessary for local execution.
+
+When developing revolver, it's useful to configure it to run entirely locally with the settings:
+* `localResourcesJson: resources.json`
+* `pretend: true`
+* `revolverRoleName: none`
```
Comment (lyricnz) [inline CONTRIBUTING.md]: maybe add a note about testing within a single account using "none" role?

**Review by lyricnz — APPROVED** [review]
LGTM
